### PR TITLE
Woopra id cookie fix

### PR DESCRIPTION
**Pull request by philosoralphter**

### CODE REVIEW

File: lib/mapper.js [hunk 1]
```diff
@@ -68,10 +69,26 @@ exports.track = function(track){
  */
 
 function cookie(message){
-  return crypto
+  var woopraCookie;
+  //use try/catch because property is 5 deep
+  try{
+    woopraCookie = message.options().context.integrations.woopra.cookie;
```
Comment (philosoralphter) [inline lib/mapper.js]: Is this the appropriate way to access the cookie on the server side?

Comment (calvinfo) [inline lib/mapper.js]: @philosoralphter you'll want to use `message.options('Woopra').cookie`, which should pick it up properly from options or integrations for woopra

File: lib/mapper.js [hunk 2]
```diff
@@ -46,7 +47,7 @@ exports.track = function(track){
   delete props.ce_id;
   delete props.ce_email;
   return extend(props, {
-    cv_id: track.userId(),
```
Comment (hankim813) [inline lib/mapper.js]: my worry is that this is seriously start breaking existing users' current woopra setup

File: lib/mapper.js (hunk 1 above)
Comment (hankim813) [inline lib/mapper.js]: I would do something like:

```js
var woopraSettings = message.options('Woopra');
if (woopraSettings && woopraSettings.cookie) return woopraSettings.cookie;
```
before the hashing

**Comment by philosoralphter** [comment]
Separating id issues from cookie issues. New PR for cookie fix.

**Pull request closed without merging**